packages/webapp/src/components/DateRangeSelector/CustomDateRangeSelector.jsx

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import clsx from 'clsx';
+import { BsChevronLeft } from 'react-icons/bs';
+import { Semibold, Underlined } from '../Typography';
+import { FromToDateContainer } from '../Inputs/DateContainer';
+import styles from './styles.module.scss';
+
+export default function CustomDateRangeSelector({
+  onBack,
+  onClear,
+  isValid,
+  validRange,
+  startDate,
+  endDate,
+  changeStartDate,
+  changeEndDate,
+  fromDateMax,
+  toDateMin,
+}) {
+  const { t } = useTranslation();
+  const inputStyle = { classes: { container: { minWidth: 'calc((100% - 24px) / 2)' } } };
+
+  return (
+    <div className={styles.customDateRangeSelector}>
+      <div className={styles.buttons}>
+        <div
```

Comment:
Oh I clearly copy pasted the wrong thing 😂 sorry about that! This is the right link https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
Sorry I wasn't sure exactly where this landed (there were a few threads on button) but I see

> I just learned default button style is reset :)

Just checking that this means this will be updated back to how it used to look?

![Screenshot 2023-10-13 at 10 25 30 AM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/d426e2e0-c962-4d9b-be3e-90442b7d2812)

Reply:
😨😨😨!!!
I guess I needed to restart storybook, it looked normal after I switched to `button`.
Thank you for catching it, I will fix it!